components/producers/producer_test.go

```diff
@@ -121,3 +123,29 @@ func TestParseMultiJSONMessages(t *testing.T) {
 	}
 	assert.Equal(t, want[0].Foo, "bar")
 }
+
+func TestGetPURLTarget(t *testing.T) {
+	target := GetPURLTarget("deb", "debian", "curl", "7.68.0", nil, "")
+	assert.Equal(t, "pkg:deb/debian/curl@7.68.0", target)
+
+	target = GetPURLTarget("bitbucket", "birkenfeld", "pygments-main", "244fd47e07d1014f0aed9c", nil, "")
+	assert.Equal(t, "pkg:bitbucket/birkenfeld/pygments-main@244fd47e07d1014f0aed9c", target)
```

Comment:
```suggestion
	require.Equal(t, "pkg:bitbucket/birkenfeld/pygments-main@244fd47e07d1014f0aed9c", target)
```

assertions package introduced `require` which allows for immediate exit and less noise, can we please use that?